server/models/User.js

```diff
@@ -0,0 +1,13 @@
+const mongoose = require("mongoose")
+
+const UserSchema = new mongoose.Schema(
```

Comment:
Alternatively, we could list productIDs here instead of customerIDs/email/whatever on the product schema. We could also do both, but redundancy isn't always great. What do you think?

Reply:
This part is the login process, which is related to what Sonny is doing. I don't want to interfere with what he has been doing, so I might as well delete/comment on this file. If we want to use it, I can merge it back in.